chia/farmer/farmer.py

```diff
@@ -608,7 +612,7 @@ async def update_cached_harvesters(self) -> bool:
                 self.log.debug(f"update_cached_harvesters update harvester: {connection.peer_node_id}")
                 cache_entry.bump_last_update()
                 response = await connection.request_plots(
-                    harvester_protocol.RequestPlots(), timeout=UPDATE_HARVESTER_CACHE_INTERVAL
+                    harvester_protocol.RequestPlots(), timeout=self.update_harvester_cache_interval
```

Comment:
it seems a bit odd to use the *cache* interval as the timeout to this request. It's not obvious that it makes sense.

Reply:
It's coming from https://github.com/Chia-Network/chia-blockchain/pull/7244.